### PR TITLE
place rails under buildings so rails connect with them - but don't ha…

### PR DESCRIPTION
**Pull request by Byteron**
…ve them be connectors

### CODE REVIEW

File: source/game/map/Map.gd [hunk 1]
```diff
@@ -401,15 +402,20 @@ func add_contruction(origin: Tile, data: ConstructionData) -> void:
...
-	rails_overlay.update_bitmask_area(tile.position)
+	_add_rail(tile)
 
 	var connector = Connector.new(tile)
 	connectors[tile.position] = connector
 
 	return connector
 
+func _add_rail(tile: Tile) -> void:
+	rails_overlay.set_cellv(tile.position, 0)
```
Comment (T4g1) [inline source/game/map/Map.gd]: Use constant instead of arbitrary values whenever possible

**Review by T4g1 — CHANGES_REQUESTED** [review]
You forget to remove rails when removing buildings

File: source/game/map/Map.gd (hunk 1 above)
Comment (Byteron) [reply source/game/map/Map.gd]: changes pushed.

**Review by T4g1 — APPROVED** [review]
lgtm, not tested, i trust you it works :)

**Pull request merged**